packages/netlify-cms-backend-git-gateway/src/GitLabAPI.ts

```diff
@@ -15,16 +14,15 @@ export default class API extends GitlabAPI {
     this.repoURL = '';
   }
 
-  authenticateRequest = async (req: ApiRequest) =>
-    unsentRequest.withHeaders(
+  withAuthorizationHeaders = async (req: ApiRequest) => {
+    const token = await this.tokenPromise();
+    return unsentRequest.withHeaders(
       {
-        Authorization: `Bearer ${await this.tokenPromise()}`,
+        Authorization: `Bearer ${token}`,
       },
       req,
     );
-
```

Comment:
To support backoff with git-gateway + gitlab I removed this and just override `withAuthorizationHeaders `